clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1083,6 +1089,13 @@ private int handleSingleMessage(
             record.getTopicPartition().getPartitionNumber(),
             partitionConsumptionStateMap.get(topicPartition.getPartitionNumber()));
       }
+    } else if (record.getKey().isDivControlMessage()) {
+      // This is a control message from the DIV topic, process it and return early.
```

Comment:
I am a bit confused about this "DIV topic" part. Do you mean follower see a DIV control message from local VT?

Reply:
Changed to:
```
// This is a global realtime topic data integrity validator object, process it and return early.
```